### ISSUE TITLE
Support for the version info

### ISSUE THREAD

**Issue by daeho-ro** [robertpsoane/ducker#27]
**Is your feature request related to a problem? Please describe.**
Can you add the version argument such as --version, -v and return the version of ducker, i.e. v0.0.3?

**Describe the solution you'd like**
Add version argument and return version info.

**Describe alternatives you've considered**

**Additional context**

**Comment by robertpsoane** [comment]
Yep I'll add that

**Comment by robertpsoane** [comment]
@daeho-ro - I've just started to pick this up and noticed that there already is a version flag - is this not working on your install?
```bash
❯ ducker --version
ducker 0.0.3
```

**Comment by daeho-ro** [comment]
OMG, it worked!

**Comment by daeho-ro** [comment]
I have added ducker into my own homebrew tap, and so I can install it now as follows.
```
brew tap daeho-ro/tap
brew install ducker
```

Once the condition for submit to the official homebrew, I will transfer it later.

**Comment by robertpsoane** [comment]
Thanks!!
Feel free to add this to the README if you want

**Comment by robertpsoane** [comment]
@daeho-ro - you might be interested to see I've added the version info to the bottom right of the terminal - this isn't in 0.0.5 but will be in 0.0.6 onwards - will indicate when there's been a new release so uses can upgrade

It's also possible to switch off this behaviour in the config

**Comment by daeho-ro** [comment]
Once a new version is released, 
```
brew lc ducker
> ducker: 0.0.3 ==> 0.0.5
```
will print the most recent version info.

After that, any brew user can create a PR to update ducker info. 
```
brew bump --formula --open-pr ducker
```

I have done it now, merged a PR and it will show the version 0.0.5.
